test/docker-worker-test

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env babel-node
+
+import program from 'commander';
+import { spawn } from 'child_process';
+import path from 'path';
+import diveSync from 'diveSync';
+
+let baseDir = path.resolve(__dirname, '..');
+let binary = `${baseDir}/node_modules/mocha/bin/mocha`;
```

Comment:
nit: I think you'll want to use `${baseDir}/node_modules/.bin/mocha`, at least that is the npm pattern... Mocha could (if they were crazy) decide to move the binary location within their package.